Source/WebInspectorUI/UserInterface/Views/ScreenshotsTimelineOverviewGraph.js

```diff
@@ -67,10 +67,9 @@ WI.ScreenshotsTimelineOverviewGraph = class ScreenshotsTimelineOverviewGraph ext
         let secondsPerPixel = this.timelineOverview.secondsPerPixel;
 
         for (let record of this._visibleRecords()) {
-            this.element.appendChild(this._imageElementForRecord.getOrInitialize(record, () => {
+            let recordElement = this.element.appendChild(this._imageElementForRecord.getOrInitialize(record, () => {
                 let imageElement = document.createElement("img");
                 imageElement.height = this.height;
```

Comment:
NIT: I realize that `this.height` is a constant, but maybe we should move this outside too in case that changes in the future 🤔

Reply:
I guess since this function gets the height dynamically the right thing to do would be to assume it could change some day.